jb/itch5/order_book_depth.hpp

```diff
@@ -1,8 +1,9 @@
-#ifndef jb_itch5_order_book_hpp
-#define jb_itch5_order_book_hpp
+#ifndef jb_itch5_order_book_depth_hpp
```

Comment:
I would not rename the class, you are just adding some functionality, it is still basically about maintaining an order book.